README.md

```diff
@@ -67,9 +67,9 @@ for the base that Ramp works in. Since the base is equal to 2^word_size, these a
 ## Future Work
 
 Ramp is currently very rough and incomplete. Broadly, there are three types Ramp aims to provide:
-integers, rationals and floats. Integers (`Int`), is present, but incomplete, the other two are not
```

Comment:
I think I probably meant this to read "Integers are present", following on from the previous sentence.